auth.go

```diff
@@ -104,6 +104,10 @@ func NewAutoAuth(login string, secret string) Authorizer {
 		return &DigestAuth{login, secret, digestParts(rs)}, nil
 	})
 
+	az.AddAuthenticator("passport1.4", func(c *http.Client, rs *http.Response, path string) (auth Authenticator, err error) {
+		return NewPassportAuth(login, secret, rs.Request.URL.String(), &rs.Header)
```

Comment:
I've not test it,  but I think `rs.Header.Get("Location")` is the parnterURL!
Because `rs.Request.URL.String()` should be the origin request, not the upcoming (redirected).

Reply:
same as above - intended